chat-widget/src/common/Constants.ts

```diff
@@ -304,6 +304,12 @@ export class NotificationPaneConstants {
     public static readonly ChromeCloseIconName = "ChromeClose";
 }
 
+export class StartChatErrorPaneConstants {
+    public static readonly DefaultStartChatErrorPaneId = "oc-lcw-start-chat-error-pane";
+    public static readonly DefaultStartChatErrorTitleText = "We are unable to load chat at this time.";
```

Comment:
we need to localize default messages?

Reply:
Localization will happen in the LCW OOB layer, the changes will be made there.